src/xinterpreter.cpp

```diff
@@ -86,161 +91,167 @@ void ExecSimpleCommand(const std::string code) {
 
 
 ///\returns true on error.
-static bool ProcessCode(clang::Interpreter& Interp, const std::string& code, std::string& err) {
```

Comment:
Can you undo the formatting changes here and leave only the relevant changes in this PR?